CONTRIBUTING.md

```diff
@@ -47,6 +47,7 @@ below:
 * Bruno P. Kinoshita (National Institute of Water and Atmospheric Research, New Zealand)
 * Tim Pillinger (Met Office, UK)
 * Ronnie Dutta (Met Office, UK)
+* Mel Hall (Met Office UK)
```

Comment:
Additionally `git shortlog -sn` suggests that the following names may also be missing·..
- Declan Valters
- Jonny Williams
- Paul Cresswell
- Roddy Sharp

Reply:
@datamel can you ok this (since it means you agree to the CLA).

Reply:
The other contributors predate the current CLA, we will have to poke them to add themselves if/when they next contribute.